cli/command/service/scale.go

```diff
@@ -10,19 +10,28 @@ import (
 	"github.com/docker/cli/cli"
 	"github.com/docker/cli/cli/command"
 	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/versions"
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
 )
 
 func newScaleCommand(dockerCli *command.DockerCli) *cobra.Command {
-	return &cobra.Command{
+	options := newServiceOptions()
```

Comment:
This shouldn't use service options (which is a huge struct).

We should create a new `scaleOptions` struct which would include just `detach` for now.